src/com/oltpbenchmark/schema/TableSchema.java

```diff
@@ -0,0 +1,60 @@
+package com.oltpbenchmark.schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TableSchema {
+    private String name;
+    private final List<Column> columns = new ArrayList<>();
+    private String primaryKey = null;
+    private String partitionKey = null;
+
+    public String name() { return name; } 
+
+    static class Builder {
+        private final TableSchema table = new TableSchema();
+        Builder name(String name) {
+            table.name = name;
+            return this;
+        }
+        Builder column(String name, String decl) {
+            table.columns.add(new Column(name, decl));
+            return this;
+        }
+        Builder primaryKey(String primaryKeyDecl) {
+            table.primaryKey = primaryKeyDecl;
+            return this;
+        }
+        Builder partitionKey(String partitionKey) {
+            table.partitionKey = partitionKey;
+            return this;
+        }
+
+        public TableSchema build() {
+            return table;
+        }
+    }
+
+    static Builder Builder() { return new Builder(); }
+
+    public void setName(String name) {
```

Comment:
setters are not needed, all fields should be configured in constructor.

Reply:
Removed this while addressing your other comment on Builder